website/content/ja/events/2024-cloud-native-sustainability-week.md

```diff
@@ -0,0 +1,67 @@
+---
+title: CNCFクラウドネイティブサステイナビリティ週間 2024
+linkTitle: Cloud Native Sustainability Week 2024
+exclude_search: true
+description: クラウドネイティブサステイナビリティ週間は、クラウドネイティブサステイナビリティをテーマに各地でCNCFコミュニティがミーティングを開催するグローバルイベントです。クラウドネイティブサステイナビリティ週間は、2024年10月の第1週に開催されます。
+weight: 94
+slug: cloud-native-sustainability-week
+aliases:
+  - /cloud-native-sustainability-week
+---
+
+<p class="mt-5 mb-5"><img src="/images/cloud-native-sustainability-week-v1-logo.webp" alt="CNCF Cloud Native Sustainability Week Banner"></p>
+
+CNCFサステイナビリティ週間は、[環境持続可能性のためのCNCF Technical Advisory Group (TAG ENV)](http://github.com/cncf/tag-env-sustainability) が [クラウドネイティブコンピューティングファウンデーション (CNCF)](http://cncf.io) と共同で毎年開催するイベントです。このイベントの主な目的は、クラウドネイティブ業界とオープンソース空間における環境持続可能性を促進することです。
+
+このグローバルイベントは、対面のミートアップとバーチャルセッションの両方を含み、世界中のローカルテックコミュニティが主催し、TAG ENVがまとめています。このようなグローバルな大陸間のコラボレーションを奨励することで、テクノロジー分野においてサステイナビリティが不可欠な要素であることを、また、時にはわずかなものであっても、環境を尊重しながら技術的・革新的な取り組みを実施することを、世界中のあらゆる人々が理解できるようになると私たちは信じています。
+
+[サステイナビリティ週間2023](https://tag-env-sustainability.cncf.io/events/cloud-native-sustainability-week-2023/)が成功を収めた後、今年の企画はすでに進行中で、より環境に優しく魅力的な取り組みへの新たな取り組みが約束されています。
```

Comment:
```suggestion
[サステイナビリティ週間2023](https://tag-env-sustainability.cncf.io/events/cloud-native-sustainability-week-2023/)での成功を受けて、今年の企画はすでに進行中です。より環境に優しく魅力的な取り組みへの新たなコミットメントが約束されています。
```

IMO: 

-  It would be easier to understand if the sentence is split into two
- The word "取り組み" is repeated, so I slightly changed